OsmAnd/src/net/osmand/plus/configmap/tracks/viewholders/SortTracksViewHolder.java

```diff
@@ -48,12 +57,31 @@ public void bindView(boolean enabled) {
 		description.setText(sortMode.getNameId());
 		imageView.setImageDrawable(app.getUIUtilities().getIcon(sortMode.getIconId(), iconColorId));
 		itemView.setEnabled(enabled);
+		AndroidUiHelper.updateVisibility(filtersCount, filters != null);
+		AndroidUiHelper.updateVisibility(filtersImageView, filters != null);
+		View sortClickableView = itemView;
+		if (filters != null) {
+			sortClickableView = sortButton;
+			itemView.setOnClickListener(null);
```

Comment:
We can set OnClickListeners on doth elements and hide sortButton if necessary, why make more complicated logic?

Reply:
we don't need to hide sort button. in case filter buttn is visible, we need to disable click on whole item. if there is no filter button, click on item acts as click on sort button